metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1147,7 +1147,7 @@ class MetalsLanguageServer(
   }
 
   private def didCompileTarget(report: CompileReport): Unit = {
-    if (!isReliableFileWatcher) {
+    if (isReliableFileWatcher) {
```

Comment:
Missing not after revertion
```suggestion
    if (!isReliableFileWatcher) {
```